frontend/src/fileSender.js

```diff
@@ -24,12 +24,6 @@ class FileSender extends EventEmitter {
         if (xhr.readyState === XMLHttpRequest.DONE) {
           resolve();
         }
-
-        if (xhr.status === 200) {
-          console.log('The file was successfully deleted.');
-        } else {
-          console.log('The file has expired, or has already been deleted.');
-        }
```

Comment:
This felt weird to kill, but wasn't really "adding" anything to the user experience.
Not sure if we should be doing some metrics logging here instead of just vanilla `console.log()`.